packages/web3-eth-accounts/src/index.js

```diff
@@ -165,17 +165,17 @@ Accounts.prototype.signTransaction = function signTransaction(tx, privateKey, ca
 
         var rawTx = RLP.decode(rlpEncoded).slice(0, 6).concat(Account.decodeSignature(signature));
 
-        rawTx[7] = trimLeadingZero(rawTx[7]);
-        rawTx[8] = trimLeadingZero(rawTx[8]);
+        rawTx[7] = makeEven(trimLeadingZero(rawTx[7]));
+        rawTx[8] = makeEven(trimLeadingZero(rawTx[8]));
```

Comment:
And are you sure passing a modified value into `RLP.decode` is not going to break anything?

Reply:
Do you mean `RLP.encode`?

It looks like it dynamically calculates the length, so I don't see any problems: https://github.com/MaiaVictor/eth-lib/blob/master/lib/rlp.js#L27

In fact, I believe leaving out 0 bytes at the beginning of an int in the RLP-encoded data is a more standard approach. (not sure if it's *required* though)